cmake/onnxruntime_rocm_hipify.cmake

```diff
@@ -3,7 +3,8 @@
 
 find_package(Python3 COMPONENTS Interpreter REQUIRED)
 
-# GLOB pattern of file to be excluded
+# Those are string patterns to exclude. Do NOT use stars such as
```

Comment:
This comment is not accurate.
https://cmake.org/cmake/help/latest/command/file.html#filesystem mentioned an example:
/dir/*.py match all python files in /dir and subdirectories

Reply:
Then perhaps the real problem is that the CI pipeline for ROCm does not build with nccl and mpi enabled.  If nccl is enabled, the files collective/sharded_moe.h and collective/sharded_moe.cc are included in the hipified list of files.

Reply:
I think so